Cabal/Cabal.cabal

```diff
@@ -341,16 +314,16 @@ test-suite package-tests
     tasty-quickcheck,
     tasty-hunit,
     QuickCheck >= 2.1.0.1 && < 2.9,
+    transformers,
```

Comment:
Wouldn't this require changes in bootstrap scripts...? Or perhaps it doesn't when it's just "test" deps?